radio/src/fonts/480x272/font_dblsize.specs

```diff
@@ -1 +1 @@
-0,4,14,28,54,76,110,139,146,160,173,193,217,228,242,251,266,289,310,331,352,375,397,420,441,464,487,497,508,532,556,580,598,613,641,665,689,715,736,757,783,808,818,832,859,880,910,935,963,987,1015,1040,1062,1087,1112,1140,1178,1206,1234,1258,1272,1287,1300,1324,1343,1356,1378,1401,1420,1443,1466,1484,1507,1529,1539,1553,1576,1586,1618,1640,1663,1686,1709,1726,1746,1764,1786,1810,1841,1865,1889,1909,1933,1940,1964,1992,2019,2037,2055,2078,2101,2124,2147,2175,2175,2175,2175,2175,2175,2175,2175,2175,2175,2175,2175,2175,2198,2221,2243,2258,2277,2305,2327,2355,2378,2403,2425,2448,2471,2491,2510,2527,2547,2571,2593,2607,2630,2652,2675,2697,2719,2750,2770,2791,2813,2837,2862,2886,2914,2942,2955,2976,3001,3029,3054,3079,3105,3130
\ No newline at end of file
+0,4,14,25,45,64,89,111,117,129,141,157,175,184,197,207,220,239,258,277,296,315,334,353,372,391,410,420,429,446,465,483,500,513,536,557,579,601,620,639,662,686,696,715,737,755,784,808,831,853,876,897,918,939,961,983,1012,1033,1054,1074,1084,1100,1110,1125,1141,1153,1171,1190,1208,1227,1245,1258,1277,1296,1305,1314,1333,1342,1371,1390,1409,1428,1447,1460,1477,1489,1508,1525,1550,1568,1585,1602,1624,1633,1651,1664,1677,1690,1703,1716,1729,1742,1755,1768,1783,1798,1813,1828,1843,1858,1873,1888,1903,1918,1933,1948,1966,1984,2002,2014,2032,2055,2073,2096,2115,2137,2156,2177,2195,2212,2230,2244,2261,2278,2296,2308,2326,2345,2364,2383,2402,2426,2441,2460,2481,2503,2524,2544,2565,2588,2600,2619,2643,2666,2688,2710,2732,2753,2777,2796,2814,2832,2851,2874,2897,2920,2938,2957,2975,2993,3011,3022,3041,3060,3077,3094,3111,3134,3156,3175,3194,3218,3241,3262,3282,3302,3320,3338,3357,3380,3403,3426
```

Comment:
perhaps those specs files could be removed now? I added them here because at the time the font was not the same on Windows and Linux

Reply:
I thought they are used by the code to know the beginning of each char.